examples/client/Locomotion/src/Components/Marker/index.js

```diff
@@ -21,7 +21,7 @@ export default ({
 }) => {
   const { chosenService } = useContext(RidePageContext);
   const { lat, lng } = stopPoint;
-  const eta = stopPoint.eta || (chosenService && chosenService.eta);
+  const eta = stopPoint.plannedArrivalTime || (chosenService && chosenService.plannedArrivalTime);
```

Comment:
do we have plannedArrivalTime on service?

Reply:
🤦‍♂️